snippets/csharp/language-reference/keywords/volatile/Program.cs

```diff
@@ -54,7 +55,7 @@ public static void Main()
 
             // Put the main thread to sleep for 1 millisecond to
```

Comment:
```suggestion
            // Put the main thread to sleep for 500 millisecond to
```

Reply:
I had changed this but had to rebase the code and must have lost it. Nice catch.